configs/swarm/client.toml

```diff
@@ -5,7 +5,7 @@ torii_url = "http://127.0.0.1:8080/"
 web_login = "mad_hatter"
 password = "ilovetea"
 
-[account]
-id = "alice@wonderland"
-public_key = "ed01207233BFC89DCBD68C19FDE6CE6158225298EC1131B6A130D1AEB454C1AB5183C0"
-private_key = { algorithm = "ed25519", payload = "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0" }
+[account] # alice
+domain_id = "wonderland"
```

Comment:
I'm not sure I understand your concern.
Do you have any specific change suggestions for this file?
Is it something other than removing the comment that mentions a specific alias?

Reply:
My concern is that: if I open <kbd>/configs/swarm</kbd> directory and run `docker compose up`, will it just work with these configs?

Reply:
I ran pytest with docker compose environment and saw the same result as one with `test_env.py` environment, so I think there are no issue specific to swarm.
One of the rest tasks in this PR is providing the same sample aliases as the cargo tests to pytest client, and have client and peer communicate in resolved format.
Does this answer your question?

Reply:
No, it doesn't =(

What I am hearing from you:

- Rust tests use sample aliases everywhere with `.parse_alias` helper. Aliases resolution doesn't have anything to do with Iroha internals. It is essentially a shorthand to create `AccountId` in a readable way on client side and then pass it to Iroha.
- ~~You set up the same aliases resolution mechanism in Pytests~~ Upon observing pytests changes, I see that you replaced account names with public keys. Thus, you are not using aliases there.
- You set up test env with `test_env.py` and run pytests, and it works.
- You also run `docker compose up` manually (I guess, with one of the swarm compose files) and it works.

Well, now am not confused... Maybe the diff of the file initially was different, but now it seems fine: client config has `account` with domain id and key pair, and the account id is a combination of public key and domain id.

The only thing remains confusing to me: "alice". Within the scope of swarm configurations, it doens't make sense. Alice exists only within test aliases. **<kbd>/configs/swarm</kbd> is a thing on its own**, not related to tests in any way. The fact that test environment relies on swarm configs is only because it is convenient so far.

I can elaborate more on that, but the request is tiny: please remote mention of Alice here.